lms/static/scripts/frontend_apps/components/ContentSelector.js

```diff
@@ -239,18 +225,18 @@ export default function ContentSelector({
           >
             Enter URL of web page or PDF
           </LabeledButton>
-          {canvasFilesEnabled && (
+          {lmsFilesEnabled && family === 'canvas' && (
             <LabeledButton
-              onClick={() => selectDialog('canvasFile')}
+              onClick={() => selectDialog('lmsFile')}
               variant="primary"
               data-testid="canvas-file-button"
             >
               Select PDF from Canvas
             </LabeledButton>
           )}
-          {blackboardFilesEnabled && (
+          {lmsFilesEnabled && family === 'BlackboardLearn' && (
             <LabeledButton
-              onClick={() => selectDialog('blackboardFile')}
+              onClick={() => selectDialog('lmsFile')}
```

Comment:
This feels like we are having to work around the fact we've collapsed different picker types into the same type?